cli/azd/pkg/project/project_config.go

```diff
@@ -58,6 +58,8 @@ const (
 	Destroying Event = "destroying"
 	// Raised after project is destroyed
 	Destroyed Event = "destroyed"
+	// Raised after project is deployed or environment setting is refreshed
+	DotnetSecretSetting Event = "dotnet secrets setting"
```

Comment:
I think we can name this `EnvironmentUpdated` and it will work for both deployed + refresh case